scripts/install/opt/seagate/cortx/motr/bin/motr_mini_prov.py

```diff
@@ -129,8 +138,28 @@ def validate_motr_rpm(self):
     sys.stdout.write(f"Checking for {MOTR_SYS_FILE}\n")
     validate_file(MOTR_SYS_FILE)
 
+def verify_lnet(self):
+    sys.stdout.write("Doing ping to nids.\n")
+    ret = lnet_self_ping(self)
+    if not ret:
+        # Check if lent is up. If not, restart lnet and try ping nid.
+        # Else, ping nid after some delay since lnet is already up.
+        if not check_services(self, ["lnet"]):
+            sys.stdout.write("lnet is not up. Restaring lnet.\n")
+            restart_services(self, ["lnet"])
+            sys.stdout.write("Doing ping to nids after 5 seconds.\n")
+            execute_command_without_exception(self, "sleep 5")
+            ret = lnet_self_ping(self)
+        else:
+            sys.stdout.write("lnet is up. Doing ping to nids after 5 seconds.\n")
+            execute_command_without_exception(self, "sleep 5")
+            ret = lnet_self_ping(self)
```

Comment:
These two commands
```
            execute_command_without_exception(self, "sleep 5")
            ret = lnet_self_ping(self)
```
are duplicated. 
They will be called in both cases.
So, we can call it outside of this "if ... else ...".

Reply:
Done 19c9afc